### ISSUE TITLE
A11y:  Can't use keyboard to navigate color areas in color picker

### ISSUE THREAD

**Issue by capeoples** [Esri/calcite-components#1406]
### Summary
In the color picker, it is currently not possible to tab to the controls that operate the color field and slider (highlighted below).
<img width="319" alt="Screen Shot 2020-12-15 at 2 12 00 PM" src="https://user-images.githubusercontent.com/30473086/102279559-74e9ae00-3ee0-11eb-9b7f-917955c2035d.png">

### Expected Behavior
To make this accessible, a user should be able to tab to the color area buttons and then use the arrow keys to control where that button is in the color field or slider. I've found a color selector component that support this keyboard behavior here: https://app.contrast-finder.org/?lang=en

Note, a non-calcite color picker was flagged for having similar a11y issues in the Level Access accessibility audit done for Enterprise in Sept 2020.

**Comment by bstifle** [comment]
@jcfranco do you have an eta on this?

**Comment by jcfranco** [comment]
@bstifle No, sir. I do not.

**Comment by paulcpederson** [comment]
@bstifle eta is whenever #1885 lands 😄

**Comment by jcfranco** [comment]
Installed. Thanks @paulcpederson for tackling this one! 🎉 

@capeoples could you help verify? It should be available for testing on `@esri/calcite-components@next` or when 1.0.0-beta.52 ships.

**Comment by jcfranco** [comment]
Verified w/ beta.52 update of https://esri.github.io/calcite-components/?path=/story/components-controls-colorpicker--simple.
